### ISSUE TITLE
[Accessibility][Screen Readers]: Edit field under Custom Picker (Document Picker) variant does not have a label. (Bug 2) 

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10821]
Breakout of issue #8655

**Comment by joschect** [comment]
This no longer repros on the main page, the document picker does have a label now. Closing this as it is no longer relevant.
